bmlt-meeting-list.php

```diff
@@ -214,6 +215,8 @@ function enqueue_backend_files($hook) {
 				wp_enqueue_script("tooltipster", plugin_dir_url(__FILE__) . "js/jquery.tooltipster.min.js", array('jquery'), "1.2", true);
 				wp_enqueue_script("spectrum", plugin_dir_url(__FILE__) . "js/spectrum.js", array('jquery'), "1.2", true);
 				wp_enqueue_script("chosen", plugin_dir_url(__FILE__) . "js/chosen.jquery.min.js", array('jquery'), "1.2", true);
+				wp_enqueue_script("selectr", plugin_dir_url(__FILE__) . "js/selectr.min.js");
```

Comment:
We already have a control that seems to be similar to this.  I'd rather avoid adding "another" javascript/css library.  Check out how "additional meetings" is being used.

Reply:
When the user changes a meeting list definition, he is effectively changing a Wordpress option.  Therefore, he needs the "manage-options" capability.  This is normally associated with admins, but at least on my local server, I can create a user, give him this capabilty, and then he is able to save.

Reply:
Theres a explorative testing tour called "back alleys"...Bread needs it.  I use bread often and intensely, and I never played with "extra meetings"...

Reply:
I use "extra meetings" on two different bread installs.